### PR TITLE
Text size and brightness preferences to show a preview under seek bar

### PR DESCRIPTION
**Pull request by amynbe**
#708

https://user-images.githubusercontent.com/1658509/103448543-ecfef480-4c92-11eb-9498-2d7abef6e58d.mp4

### CODE REVIEW

File: app/src/main/java/com/quran/labs/androidquran/ui/translation/TranslationAdapter.kt [hunk 1]
```diff
@@ -432,7 +432,7 @@ internal class TranslationAdapter(private val context: Context,
 
   companion object {
     private val USE_UTHMANI_SPAN = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
-    private const val ARABIC_MULTIPLIER = 1.4f
```
Comment (amynbe) [inline app/src/main/java/com/quran/labs/androidquran/ui/translation/TranslationAdapter.kt:435]: making this public so SeekBarTextSizePreference can show an accurate preview of arabic text size

File: app/src/main/res/layout-v21/seekbar_pref.xml [hunk 2]
```diff
@@ -51,6 +51,28 @@
         />
   </LinearLayout>
```
Comment (amynbe) [inline app/src/main/res/layout-v21/seekbar_pref.xml:53]: Adding these 2 text views under the seekbar, one for arabic and one for the translated language.
They are hidden by default in SeekBarPreference.java and the overriding classes opt in to show them and do something with them (i.e apply text size or brightness on them).

**Review by amynbe — COMMENTED** [review]
Have tried an approach with preview text under the seek bar.

File: app/src/main/res/layout-v21/seekbar_pref.xml [hunk 3]
```diff
@@ -51,6 +51,28 @@
...
+        android:layout_height="wrap_content"
+        android:text="@string/prefs_preview"
+        android:gravity="center"
+        android:layout_weight="1"/>
+
+    <TextView
+        android:id="@+id/arabic_preview"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="معاينة"
```
Comment (ahmedre) [inline app/src/main/res/layout-v21/seekbar_pref.xml:70]: should be `@string/prefs_preview` and we should add to `values-ar/strings.xml`

File: app/src/main/res/layout/seekbar_pref.xml [hunk 4]
```diff
@@ -53,6 +53,28 @@
...
+        android:layout_height="wrap_content"
+        android:text="@string/prefs_preview"
+        android:gravity="center"
+        android:layout_weight="1"/>
+
+    <TextView
+        android:id="@+id/arabic_preview"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="معاينة"
```
Comment (ahmedre) [inline app/src/main/res/layout/seekbar_pref.xml:72]: same as above

**Review by ahmedre — APPROVED** [review]
really sorry for the very late reply - this looks great masha'Allah! minor comment about the Arabic string, but let's merge this in sha' Allah

File: app/src/main/res/layout-v21/seekbar_pref.xml (hunk 3 above)
Comment (ahmedre) [reply app/src/main/res/layout-v21/seekbar_pref.xml:70]: oh - i see why now - will add another one just for this and make it non-translatable

**Comment by ahmedre** [comment]
jazakumAllah khairan - merged here - 25a8d74f00c8d5adaf0c3657b4f6fa902b2d6ed4

**Pull request closed without merging**